setup.sql

```diff
@@ -0,0 +1,149 @@
+CREATE DATABASE novels;
```

Comment:
You can delete this file. We used .sql files as a stand in until we learned migrations. Real world projects would use migrations from the beginning and never have .sql files.